### ISSUE TITLE
Install fails due to missing files

### ISSUE THREAD

**Issue by rnhmjoj** [MoonchildProductions/Pale-Moon#826]
Hi, I have been trying to package Pale Moon for NixOS. Here is the progress made so far: https://github.com/NixOS/nixpkgs/pull/21726. The build succeeds and the browser seems to run ok but `mach install` fails with these errors:

```
Error: /tmp/nix-build-palemoon-27.0.3.drv-0/palemoon-src/obj-x86_64-unknown-linux-gnu/browser/installer/package-manifest:197: Missing file(s): bin/browser/components/status4evar.xpt
Error: /tmp/nix-build-palemoon-27.0.3.drv-0/palemoon-src/obj-x86_64-unknown-linux-gnu/browser/installer/package-manifest:349: Missing file(s): bin/browser/components/status4evar.js
Error: /tmp/nix-build-palemoon-27.0.3.drv-0/palemoon-src/obj-x86_64-unknown-linux-gnu/browser/installer/package-manifest:350: Missing file(s): bin/browser/components/status4evar.manifest
```

Do you have any idea why?

**Comment by wolfbeast** [comment]
Do not use "mach install"; if you want to package, use "mach package"

**Comment by rnhmjoj** [comment]
`mach install` is ok for building Nix packages, besides `mach package` gives the same error.
By removing the lines mentioning `status4evar` in the manifest with

    sed -i /status4evar/d browser/installer/package-manifest.in

the install went through. I don't know if this broke something though.

**Comment by mattatobin** [comment]
Well you have just busted the statusbar components by not having them packaged/installed. The real question is why are they not where they are supposed to be in the dist/bin folder that package-manifest expects them to be.

**Comment by wolfbeast** [comment]
> `mach install` is ok for building Nix packages,

Our source tree isn't like other *nix packages. It's different because it is multi-platform and installing directly from the build tree is at the very least untested because it is not the intended way of installing the browser, and may not work.

@mattatobin : would MOZ_BROWSER_STATUSBAR ever be false, unless explicitly disabled in configure?

**Comment by wolfbeast** [comment]
The above commit should take care of this in `--disable-browser-statusbar` situations.

**Comment by wolfbeast** [comment]
@rnhmjoj could you verify that the current trunk packages properly with the statusbar option disabled?

**Comment by rnhmjoj** [comment]
Sorry for the delay but I have bee quite busy.
> Also, can you for the sake of interest give us your mozconfig.

You can see the mozconfig in the [package source](https://github.com/NixOS/nixpkgs/blob/46eecaf0d4ce986c3d4e9ff52c3c512e45c7a6c3/pkgs/applications/networking/browsers/palemoon/default.nix#L59).

> Our source tree isn't like other *nix packages. It's different because it is multi-platform and installing directly from the build tree is at the very least untested because it is not the intended way of installing the browser, and may not work.

Well, I mostly copied the expression already used [here](https://github.com/Axiomatic-/Pale-Moon-Nix/blob/master/release.nix) to make a build environment/release and I haven't found any particular problem so far.

> could you verify that the current trunk packages properly with the statusbar option disabled?

I can confirm it's solved by 515f69a. Thank you.
